tests/test_manager.py

```diff
@@ -116,38 +76,43 @@ def test_scaling_data(mock_plot, scaling_data):
     and that the plotting function is called correctly.
     """
     paths, ncpus, datasets = scaling_data
-    config_prof = MockProfilingManager(paths, ncpus, datasets)
+    manager = MockProfilingManager(paths, ncpus, datasets)
 
-    config_prof.parse_scaling_data()
+    manager.parse_scaling_data()
 
-    assert set(config_prof.data.keys()) == {"component"}
-    assert set(config_prof.data["component"].dims) == {"ncpus", "region"}, (
+    assert set(manager.data.keys()) == {"component"}
+    assert set(manager.data["component"].dims) == {"ncpus", "region"}, (
         "Dataset should have dimensions 'ncpus' and 'region'!"
     )
-    assert config_prof.data["component"].sizes["ncpus"] == 3, "Dataset should have 2 values for 'ncpus'!"
-    assert config_prof.data["component"].sizes["region"] == 2, "Dataset should have 3 values for 'region'!"
+    assert manager.data["component"].sizes["ncpus"] == 3, "Dataset should have 2 values for 'ncpus'!"
+    assert manager.data["component"].sizes["region"] == 2, "Dataset should have 3 values for 'region'!"
 
-    assert set(config_prof.data["component"].data_vars) == {count, tavg}, (
-        "Dataset should have data_vars for each metric!"
+    assert manager.data["component"]["ncpus"].values.tolist() == [1, 2, 4], (
+        "Dataset should have correct 'ncpus' coordinate values!"
```

Comment:
```suggestion
        "Dataset should have correct 'ncpus' coordinate values (in sorted order)!"
```

Or some variant to show that the ncpus numbers should be in sorted order. Particularly, because the test setup is intentionally in not-sorted order. 

Another option could be to check against `sorted(ncpus)` - so that the test still works even if we change the ncpus list in the future